modules/api-resources/api-resources-full/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -125,23 +125,24 @@
     </jaxrs:server>
     <jaxrs:server id="users" address="/users/v1">
         <jaxrs:serviceBeans>
+            <bean class="org.wso2.carbon.identity.rest.api.user.application.v1.MeApi"/>
```

Comment:
why we added application management apis?

Reply:
This is a alphabetic re-ordering